webapp/src/components/checklist/checklist_list.tsx

```diff
@@ -58,13 +51,25 @@ interface Props {
     playbookRun?: PlaybookRun;
     playbook?: Loaded<FullPlaybook>;
     enableFinishRun: boolean;
+    isReadOnly: boolean;
+    checklistsCollapseState: Record<number, boolean>;
+    onChecklistCollapsedStateChange: (checklistIndex: number, state: boolean) => void;
+    onEachChecklistCollapsedStateChange: (state: Record<number, boolean>) => void;
```

Comment:
This function is called when the collapsed state of every checklist is changed at once, right? If that's correct, I proposeto use `onEveryChecklist` or `onAllChecklsits` instead of `Each`. At least for me the `Each` word threw me off.
```suggestion
    onEveryChecklistCollapsedStateChange: (state: Record<number, boolean>) => void;
```

Reply:
> Also, a more general question: why does the Better Playbook Editor now store the state locally instead of using yet another key for the store? Does that play well with different users viewing the playbook at the same time and one of them checking and unchecking tasks?

@agarciamontoro  `BPE` never supported caching checklist state in the redux, so I used the local store. If storing the state in the redux makes sense, I can add it, it's just a few lines of code. I don't have a strong opinion regarding this. For `RHS` and `RD`, it's more clear for me because users switch between runs very often and it's important to save the state.

Reply:
@gbochora Oh, wait, I completely mixed up two things: whether the checklist is collapsed or not and the actual state of the tasks (which is not modified in the playbook editor, sorry :facepalm:). If they didn't add that to redux then let's skip it for now. Thanks for the explanation and sorry for the misunderstanding!